Jenkinsfile

```diff
@@ -340,96 +352,103 @@ def listCivihrGitRepoPath() {
  */
 def listCivihrExtensions() {
   return [
-    [
+    hrjobroles: [
       name: 'Job Roles',
-      shortName: 'hrjobroles',
       folder: 'com.civicrm.hrjobroles',
       hasJSTests: true,
+      hasJSPackages: false,
```

Comment:
cc @davialexandre

Reply:
It's probably possible to do that, but I would prefer to avoid adding too much logic to this file. There are a lot of limitations on what it's possible to do with Groovy on a Jenkinsfile and keeping things simple help us avoid some of these limitations